certifikit-cli/build.gradle

```diff
@@ -84,8 +81,8 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
 graal {
   mainClass(application.mainClassName)
   outputName("cft")
-  graalVersion('21.3.0')
-  javaVersion('11')
+  graalVersion('22.0.0.2')
```

Comment:
nit: these probably want to be a separate PR ? Totally fine here for now though.